dist/index.js

```diff
@@ -7311,7 +7311,7 @@ module.exports = eval("require")("encoding");
 /***/ ((module) => {
 
 "use strict";
-module.exports = JSON.parse('{"name":"codecov-action","version":"1.5.2","description":"Upload coverage reports to Codecov from GitHub Actions","main":"index.js","scripts":{"lint":"eslint src/*.*","test":"yarn run test-script && yarn run test-calculator && yarn run test-coverage","test-calculator":"jest --testPathPattern=demo/calculator/ --coverage --coverageDirectory=coverage/calculator","test-coverage":"jest --testPathPattern=demo/coverage-test/ --coverage --coverageDirectory=coverage/coverage-test","test-script":"jest --testPathPattern=src/ --coverage --coverageDirectory=coverage/script","build":"ncc build src/index.ts"},"repository":{"type":"git","url":"git+https://github.com/codecov/codecov-action.git"},"keywords":[],"author":"Ibrahim Ali","license":"MIT","bugs":{"url":"https://github.com/codecov/codecov-action/issues"},"homepage":"https://github.com/codecov/codecov-action#readme","dependencies":{"@actions/core":"^1.4.0","@actions/exec":"^1.1.0","@actions/github":"^5.0.0"},"devDependencies":{"@types/jest":"^26.0.23","@types/node":"^14.17.3","@typescript-eslint/eslint-plugin":"^4.26.1","@typescript-eslint/parser":"^4.26.1","@vercel/ncc":"^0.28.6","eslint":"^7.28.0","eslint-config-google":"^0.14.0","jest":"^26.6.3","jest-junit":"^12.2.0","ts-jest":"^26.5.6","typescript":"^4.3.2","yarn":"^1.22.10"}}');
+module.exports = {"i8":"1.5.2"};
```

Comment:
package.json was previously embedded in its entirety on [line 7314](https://github.com/codecov/codecov-action/blob/4463da3901eda7a7e4c92f2ada425a21369ada94/dist/index.js#L7314).